ts_js_cpp_c/node-addon-api/.gitignore

```diff
@@ -0,0 +1 @@
+build
```

Comment:
This is also already in https://github.com/nodejs/node-addon-examples/blob/187598906a0cd80d4d57ecdf43ac7114de10100f/.gitignore#L1

Reply:
@gengjiawen PR updated :+1: